software/nosql/src/main/java/org/apache/brooklyn/entity/nosql/riak/RiakNode.java

```diff
@@ -125,14 +125,28 @@
 
     AttributeSensor<String> RIAK_NODE_NAME = Sensors.newStringSensor("riak.node", "Returns the riak node name as defined in vm.args");
 
-    // these needed for nodes to talk to each other, but not clients (so ideally set up in the security group for internal access)
-    PortAttributeSensorAndConfigKey HANDOFF_LISTENER_PORT = new PortAttributeSensorAndConfigKey("handoffListenerPort", "Handoff Listener Port", "8099+");
-    PortAttributeSensorAndConfigKey EPMD_LISTENER_PORT = new PortAttributeSensorAndConfigKey("epmdListenerPort", "Erlang Port Mapper Daemon Listener Port", "4369");
-    PortAttributeSensorAndConfigKey ERLANG_PORT_RANGE_START = new PortAttributeSensorAndConfigKey("erlangPortRangeStart", "Erlang Port Range Start", "6000+");
-    PortAttributeSensorAndConfigKey ERLANG_PORT_RANGE_END = new PortAttributeSensorAndConfigKey("erlangPortRangeEnd", "Erlang Port Range End", "7999+");
+    /*
+     * Needed for nodes to talk to each other, but not clients, so ideally set up in the security group for internal access and configured here.
+     * If {@link #CONFIGURE_INTERNAL_NETWORKING} is set, then a location customizer will be added to confiure the security group dynamically.
+     */
+
+    @SetFromFlag("handoffListenerPort")
+    ConfigKey<Integer> HANDOFF_LISTENER_PORT = ConfigKeys.newIntegerConfigKey("riak.handoff.port", "Handoff Listener Port", 8099);
```

Comment:
This is no longer published as a sensor, which changes backwards compatibility. It probably won't affect anyone, but you can instead use:

    @SetFromFlag("handoffListenerPort")
    AttributeSensorAndConfigKey<Integer, Integer> HANDOFF_LISTENER_PORT = ConfigKeys.newSensorAndConfigKey(
            Integer.class,
            "riak.handoff.port",
            "Handoff Listener Port", 
            8099);

Probably worth it, just in case.